Neos.ContentRepository/src/Feature/NodeRemoval/Command/RestoreNodeAggregateCoverage.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Feature\NodeRemoval\Command;
+
+use Neos\ContentRepository\DimensionSpace\DimensionSpace\DimensionSpacePoint;
+use Neos\ContentRepository\SharedModel\Node\OriginDimensionSpacePoint;
+use Neos\ContentRepository\SharedModel\Workspace\ContentStreamIdentifier;
+use Neos\ContentRepository\SharedModel\Node\NodeAggregateIdentifier;
+use Neos\ContentRepository\Feature\Common\RebasableToOtherContentStreamsInterface;
+use Neos\ContentRepository\Feature\Common\MatchableWithNodeAddressInterface;
+use Neos\ContentRepository\SharedModel\NodeAddress;
+use Neos\ContentRepository\SharedModel\User\UserIdentifier;
+use Neos\Flow\Annotations as Flow;
+
+#[Flow\Proxy(false)]
+final class RestoreNodeAggregateCoverage implements
```

Comment:
can we add some more detailed docs what the user can expect when running this command? especially why two DSPs are given is not really obvious to me (DSP would be enough, no?)